src/engine/qcommon/common.cpp

```diff
@@ -912,25 +912,28 @@ void Com_Frame()
 		{
 			if ( com_minimized->integer && maxfpsMinimized.Get() > 0 )
 			{
-				minMsec = 1000 / maxfpsMinimized.Get();
+				minMsec = 1000 / std::min(maxfpsMinimized.Get(), 333);
 			}
 			else if ( com_unfocused->integer && maxfpsUnfocused.Get() > 0 )
 			{
-				minMsec = 1000 / maxfpsUnfocused.Get();
+				minMsec = 1000 / std::min(maxfpsUnfocused.Get(), 333);
 			}
 			else if ( maxfps.Get() > 0 )
 			{
-				minMsec = 1000 / maxfps.Get();
+				minMsec = 1000 / std::min(maxfps.Get(), 333);
 			}
 			else
 			{
-				minMsec = 1;
+				minMsec = 3;
 			}
```

Comment:
Why not just doing `minSec = std::min(minSec, 3);` after all this group of `if`s?

Reply:
See [this other comment](https://github.com/DaemonEngine/Daemon/pull/595#issuecomment-1058405894) for better suggestion.